generator/test_case_generator.go

```diff
@@ -31,14 +31,16 @@ type matrixRow struct {
 	TerraformDir        string `json:"terraform_dir"`
 	UseSSM              bool   `json:"useSSM"`
 	ExcludedTests       string `json:"excludedTests"`
+	RunMockServer       bool   `json:"run_mock_server"`
 	MetadataEnabled     string `json:"metadataEnabled"`
 }
 
 type testConfig struct {
 	// this gives more flexibility to define terraform dir when there should be a different set of terraform files
```

Comment:
nit - the comment doesn't seem directly tied to `testDir`. I see it was like this before, but just wanted to point out that it looks kind of confusing to see this group of vars that aren't all related to terraform